apps/cookbook/src/app/showcase/card-showcase/card-showcase.component.ts

```diff
@@ -31,10 +31,18 @@ export class CardShowcaseComponent {
       defaultValue: '',
       type: ['string'],
     },
+    {
+      name: 'variant',
+      description:
+        'Sets the variant of a card. The recommended is "elevated" which is also the default',
+      defaultValue: 'elevated',
+      type: ['elevated', 'flat', 'outlined'],
+    },
     {
       name: 'flat',
-      description: '(Optional) Remove elevation on the card. Default elevation = 2, flat = 0.',
-      defaultValue: '',
+      description:
+        'Deprecation warning: use "variant" instead. \n(Optional) Remove elevation on the card. Default elevation = 2, flat = 0.',
```

Comment:
Should we consider removing the `flat` prop from the API docs in this PR, as we don't wan't to promote it's use anyhoo? 🤔